.pipelines/v2/templates/job-build-project.yml

```diff
@@ -233,10 +254,16 @@ jobs:
     inputs:
       solution: '**\HostsUILib.csproj'
       vsVersion: 17.0
-      ${{ if eq(parameters.useVSPreview, true) }}:
-        msbuildArgs: /p:CIBuild=true;NoBuild=true -t:pack /bl:$(LogOutputDirectory)\build-hosts.binlog /p:NoWarn=NU5104
+      ${{ if or(eq(parameters.useVSPreview, true), eq(parameters.useLatestWinAppSDK, true)) }}:
```

Comment:
Oh, with this, we can make it as single line then.
Since I see there is no security concern for "making up the build parameters", as the result, we can directly like WITHOUT IF ELSE
msbuildArgs: >-
          /p:CIBuild=true;NoBuild=true -t:pack
          /bl:$(LogOutputDirectory)\build-hosts.binlog
          $(MSBuildArgs.UseVSPreview)
          $(RestoreAdditionalProjectSourcesArg)